python/servo/testing_commands.py

```diff
@@ -111,8 +111,9 @@ def test_unit(self, test_name=None, component=None, package=None):
         self.ensure_bootstrapped()
 
         return 0 != subprocess.call(
-            ["cargo", "test", "-p", "unit_tests"]
-            + test_name, env=self.build_env(), cwd=self.servo_crate())
+            ["cargo", "test", "-p", "gfx_tests", "net_tests", "script_tests",
+                "style_tests", "util_tests"] + test_name,
```

Comment:
This doesn’t do what you’d expect. Because of https://github.com/rust-lang/cargo/issues/1528, you currently have to run `cargo test` for each crate separately.

Also, I’d prefer to have the list of crates not hard-coded here, but discovered with `os.listdir` and `os.path.isdir`.